modules/cms/src/frontend/views/commands/block/create_block.php

```diff
@@ -29,7 +29,7 @@ class <?= $className; ?> extends PhpBlock
      * @var boolean Choose whether block is a layout/container/segmnet/section block or not, Container elements will be optically displayed
      * in a different way for a better user experience. Container block will not display isDirty colorizing.
      */
-    public $isContainer = true;
+    public $isContainer = false;
```

Comment:
@dev7ch When the wizzard ask for creating a container, this should be enable. Oterhwise its hidden by default which menas $isContainer = false. 

Short: this must be `true`